src/Maestro/maestro-angular/.editorconfig

```diff
@@ -0,0 +1,11 @@
+# Editor configuration, see https://editorconfig.org
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Comment:
This 2 is... pretty narrow. And conflicts with the style of pretty much all files everywhere. C# is more indented in general than TS, and we use 4 there.

Reply:
This is angular default, and all the templates generate files with this indent_size. I'm not going to fight with it.